pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+				args = convertAnyHexToASCII(args, network);
+				if (args?.amount) {
+					requested = args.amount;
+				} else {
+					const calls = args.calls;
+
+					if (calls && Array.isArray(calls) && calls.length > 0) {
+						calls.forEach((call) => {
+							if (call && call.amount) {
+								requested += BigInt(call.amount);
+							}
+							if (call && call?.value?.amount) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)